workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/hooks/useCart.ts

```diff
@@ -165,8 +163,8 @@ const useCart = ({
 		removeCart,
 		removeFromCart,
 		setCart,
-		updateCart: headlessCommerceDeliveryCart.updateCart.bind(
-			headlessCommerceDeliveryCart
+		updateCart: HeadlessCommerceDeliveryCart.updateCart.bind(
+			HeadlessCommerceDeliveryCart
```

Comment:
This specific case of bind only is needed when working with class instance and not static methods, thank you anyway :D

```suggestion
		updateCart: HeadlessCommerceDeliveryCart.updateCart
```

Reply:
Thanks, @kevenleone! Done.